### ISSUE TITLE
Units

### ISSUE THREAD

**Issue by rameyer** [mladenovak/interferopy#15]
Include astropy.units in interferopy!

**Comment by lboogaard** [comment]
Agreed could be nice, but will take time as we have to do it carefully.
Are we sure this does not slow things down?

**Comment by rameyer** [comment]
I'm not sure - I need to check. Also it would be nice to have an internal "beam" unit, which I'm not sure how to do properly.

**Comment by lboogaard** [comment]
Related and perhaps interesting is: https://radio-beam.readthedocs.io/en/latest/ (also used by `spectral cube`)

**Comment by rameyer** [comment]
That's exactly what is needed! This plus the standard units in astropy.units should be enough.
